modules/caddyhttp/fileserver/matcher.go

```diff
@@ -181,15 +181,15 @@ func (MatchFile) CELLibrary(ctx caddy.Context) (cel.Library, error) {
 			root = values["root"][0]
 		}
 
-		var try_policy string
+		var tryPolicy string
 		if len(values["try_policy"]) > 0 {
 			root = values["try_policy"][0]
```

Comment:
Oh I think there's actually a bug here, this should be `tryPolicy = values["try_policy"][0]`

Reply:
does this commit address the issue?

https://github.com/caddyserver/caddy/pull/5712/commits/6f1286a630f226e5f679db3f6cd82b55cdc9e31b